portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -60,12 +60,48 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be edited"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeEdited {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131222 CanBeEdited pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```

Comment:
You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?

Reply:
Yes, I pushed the test with this change